src/main/java/com/ts/tserp/model/Batch_Announcement.java

```diff
@@ -0,0 +1,73 @@
+package com.ts.tserp.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity(name = "batch_announcement")
+public class Batch_Announcement {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private int id;
+
+	private String course_title;
+
+	private String start_date;
+
+	private double start_time;
+
+	private String mode;
+
+	private String trainer_name;
+
```

Comment:
You can also remove these getters, setters & constructors by using **lombok** dependency. It will reduce the number of lines of code.